health-and-fitness-organizer/src/index.css

```diff
@@ -1,13 +1,13 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
```

Comment:
so these fonts seem fairly random.  usually the max number of fonts you see in a font-family definition is 3.  it would go 

font-family: "specific font", moreGeneralFontName, generic-font-family. 

If you have to use that long a font definition, you could break the font definition into a variable just to clean up the code a little

`:root {
  --system-ui: system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
}

.element {
  font-family: var(--system-ui);
}

Reply:
this is some more code that is include in create-react-app. Since it's just fonts, I will delete it